### PR TITLE
trigger doc rebuild

### PR DESCRIPTION
**Pull request by mattdurham**
Want to trigger a doc rebuild

### CODE REVIEW

**Review by erikbaranowski — APPROVED** [review]
LGTM

**Comment by mattdurham** [comment]
Maybe we dont publish for RCs so hold off on this

**Comment by erikbaranowski** [comment]
I won't merge. I'll leave it to you when/if you want to

**Pull request closed without merging**
